tests/unit/presigned_get_object_test.py

```
@@ -63,3 +65,19 @@ def test_can_include_response_headers(self):
         self.assertIn('inline', r)
         self.assertIn('test.pdf', r)
         self.assertIn('application%2Fpdf', r)
+
+    @mock.patch('urllib3.PoolManager')
+    def test_region_is_used(self, mock_connection):
```

Pylint (4.0.6)

[convention] Missing function or method docstring

(C0116)